operator/README.md

```diff
@@ -1,2 +1,92 @@
-# pmem-csi-operator
-Operator for PMEM-CSI
+# PMEM-CSI-Operator
+
+Kubernetes operator for [PMEM-CSI](https://github.com/intel/pmem-csi) driver deployment. Using this operator one can easily deploy and manage PMEM-CSI driver in a Kubernetes cluster.
+
+The PMEM-CSI operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+
+## PMEM-CSI Deployment custom resource definition(CRD)
+
+Current PMEM-CSI Deployment object supports below API:
+
+### Deployment
+
+|Field | type | Description |
+|---|---|---|
+| apiVersion | string  | `pmem-csi.intel.com/v1alpha1`|
+| kind | string | `Deployment`|
+| metaData | [ObjectMeta](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status) | Standard objects metadata. |
```

Comment:
We need to document whether we support more than one deployment.

I think we should, in which case the CSI driver name also has to be made configurable. It may be useful at least for testing.

Reply:
We could specify that the name of the deployment is used as CSI driver name. Then there's no risk of getting two deployments with conflicting driver names. OTOH, a user may prefer a shorter name for the deployment.

I'm undecided...

Reply:
The name of the deployment is relevant, only if we plan to support more than one driver deployments, which is undecided by myself.

Reply:
So we need to decide it before we can proceed with the implementation...

Reply:
Let's support multiple deployments, but with a warning that these deployments must not run multiple drivers on the same node. Later we can investigate whether that can be supported, too.

This implies that the API needs a field for the driver name, and that field has to be unique for all deployments.

Reply:
Added "warning" for multi deployments.